konlpy/tag/_twitter.py

```diff
@@ -66,11 +73,30 @@ def phrases(self, phrase):
         """Phrase extractor."""
 
         return [p for p in self.jki.phrases(phrase).toArray()]
-    
+
     def normalize(self, phrase):
         text = self.jki.normalize(phrase)
         return text
 
+    def __spacing(self, sentence: str, pos_result: list):
```

Comment:
It should fit 3.5 version. (in build system, we still use 3.5 for compatibility)
I think 3.5 doesn't support type hint